service-api/src/main/java/greencity/dto/econewscomment/UpdateEcoNewsCommentDtoRequest.java

```diff
@@ -0,0 +1,18 @@
+package greencity.dto.econewscomment;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class UpdateEcoNewsCommentDtoRequest {
+    @NotBlank(message = "The text of comment can not be empty")
+    @Size(min = 1, max = 100)
+    public String text;
```

Comment:
You can put these annotations on the field itself and not create a DTO for one field.